bears/stylus/StylintBear.py

```diff
@@ -0,0 +1,219 @@
+        :param allow_leading_zero:
+            Enforce or disallow unnecessary leading zeroes on decimal points.
+        :param set_max_errors:
+            Set 'max' number of Errors.
+        :param set_max_warnings:
+            Set 'max' number of Warnings.
+        :param mixed_spaces_and_tabs:
+            Returns ``true`` if mixed spaces and tabs are found. If a number is
+            passed to indent_size, it assumes soft tabs (ie, spaces)
+            , and if ``false`` is passed to indentPref it assumes hard tabs.
+        :param set_naming_convention:
+            Enforce a particular naming convention when declaring classes, ids,
+            and variables. Throws a warning if you don't follow the convention.
+        :param set_strict_naming_convention:
+            By default, ``set_naming_convention`` only looks at variable names.
+            If ``set_strict_naming_convention`` is set to true,
+            ``set_naming_convention`` will  also look at class and id names.
+        :param allow_none_keyword:
+            If ``always`` check for places where ``none`` used instead of 0. If
+            ``never`` check for places where 0 could be used instead of none.
+        :param check_no_important_keyword:
+            If ``true``, show warning when ``!important`` is found.
+        :param allow_spaces_inside_parens:
+            Enforce or disallow use of extra spaces inside parens.
+        :param allow_placeholder:
+            Enforce extending placeholder vars when using ``@extend(s)``.
+        :param prefix_vars_with_dollar:
+            Enforce use of ``$`` when defining a variable. In Stylus using a
+            ``$`` when defining a variable is optional, but is a good idea
+            if you want to prevent ambiguity. Not including the ``$`` sets up
+            situations where you wonder: "Is this a variable or a value?"
+            For instance: padding ``$default`` is easier to understand than
+            padding ``default``.
+        :param allow_semicolons:
+            Enforce or disallow semicolons.
+        :param sort_order:
+            Enforce a particular sort order when declaring properties. Throws
+            a warning if you don't follow the order. If set to ``false``, allow
+            any order.
+        :param allow_stacked_properties:
+            No one-liners. Enforce putting properties on new lines.
+        :param check_valid_property:
+            Check that a property is valid CSS or HTML.
+        :param allow_zero_units:
+            Looks for instances of ``0px``. You don't need the px. Checks all
+            units, not just px.
+        :param z_index_normalize:
+            Enforce some (very) basic z-index sanity. Any number passed in
+            will be used as the base for your z-index values. Throws an error
+            if your value is not normalized.
+        :param stylint_config:
+            The location of the ``.stylintrc`` config file.
```

Comment:
> Documentation:
    Bear description shall be provided at class level.
    If you document your additional parameters inside ``create_arguments``,
    ``generate_config`` and ``process_output``, beware that conflicting
    documentation between them may be overridden. Document duplicated
    parameters inside ``create_arguments`` first, then in ``generate_config``
    and after that inside ``process_output``.

So I was actually wrong, it should be only in `create_arguments`^^

See http://api.coala.io/en/latest/_modules/coalib/bearlib/abstractions/Linter.html